Changes

```diff
@@ -107,6 +107,10 @@ Working version
 - #10349: Fix destroyed_at_c_call on RISC-V
   (Mark Shinwell, review by Nicolás Ojeda Bär)
 
+- #1400: Add an optional invariants check on Cmm, which can be activated
+  with the -dcmm-invariants flag
+  (Vincent Laviron)
```

Comment:
this shoud probably have more people as reviewers (I guess at least Stephen, probably David and Sébastien as well).